pom.xml

```diff
@@ -97,13 +97,10 @@
   </distributionManagement>
 
   <repositories>
-    <repository>
-      <id>terracotta-snapshots</id>
-      <url>http://www.terracotta.org/download/reflector/snapshots</url>
-    </repository>
     <repository>
       <id>terracotta-releases</id>
       <url>http://www.terracotta.org/download/reflector/releases</url>
```

Comment:
It may make sense to take this opportunity to change http://www.terracotta.org/download/reflector/releases to https://repo.terracotta.org/maven2

Reply:
@akomakom  If that's that it should be, certainly.  Should it be changed _universally_ ... in all projects I touch?

Reply:
Updated

Reply:
> @akomakom If that's that it should be, certainly. Should it be changed _universally_ ... in all projects I touch?

I would say yes, as the opportunity arises.